kustomize/commands/localize/localize.go

```diff
@@ -62,6 +69,31 @@ kustomize localize https://github.com/kubernetes-sigs/kustomize//api/krusty/test
 			if err != nil {
 				return errors.Wrap(err)
 			}
+
+			if !f.noVerify {
+				originalBuild, err := runBuildCmd(buildBuffer, buildCmd, args.target)
+				if err != nil {
+					return errors.Wrap(err)
+				}
+
+				buildDst := dst
+				if f.scope != "" && f.scope != args.target {
+					buildDst = filepath.Join(dst, filepath.Base(args.target))
+				}
+
+				localizedBuild, err := runBuildCmd(buildBuffer, buildCmd, buildDst)
+				if err != nil {
+					return errors.Wrap(err)
+				}
+
+				if localizedBuild == originalBuild {
+					log.Printf("VERIFICATION SUCCESS: `kustomize build` for %s and %s are the same after localization.\n", args.target, dst)
+				} else {
+					log.Println(copyutil.PrettyFileDiff(originalBuild, localizedBuild))
```

Comment:
```suggestion
					log.Println(copyutil.PrettyFileDiff(originalBuild, localizedBuild))
```

Curious: Is it intentional to add this log statement and explicitly mention the diff?

Reply:
Yes! This is what the KEP outlined:
```
The command indicates success if the outputs match and throws an error with a diff summary otherwise. 
``` 

https://github.com/kubernetes-sigs/kustomize/blob/master/proposals/22-04-localize-command.md